# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/blindcoder/nethack/hotfixes.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

This fixes a few compiler and linker errors.

don't ask... please... just don't ask
blindcoder
--- ./win/Qt/qt_win.cpp	2003-06-08 20:56:11.000000000 +0200
+++ ./win/Qt/qt_win.cpp	2003-06-08 14:27:30.000000000 +0200
@@ -47,6 +47,8 @@ extern "C" {
 // enough, and we undefine NetHack macros which conflict with Qt
 // identifiers.
 
+#include <math.h>
+
 #define alloc hide_alloc // avoid treading on STL symbol
 #define lock hide_lock // avoid treading on STL symbol
 #ifdef _MSC_VER
--- ./src/Makefile 2003-06-08 20:56:11.000000000 +0200
+++ ./src/Makefile 2003-06-08 14:27:30.000000000 +0200
@@ -147,1 +147,1 @@
-	winmisc.o winstat.o wintext.o winval.o tile.o
+	winmisc.o winstat.o wintext.o winval.o
@@ -179,1 +179,1 @@
-WINQTOBJ = qt_win.o qt_clust.o qttableview.o tile.o
+WINQTOBJ = qt_win.o qt_clust.o qttableview.o
@@ -191,1 +191,1 @@
-	gnyesno.o gnworn.o tile.o
+	gnyesno.o gnworn.o
